pkg/auth/login/login.go

```diff
@@ -53,6 +52,7 @@ func (a *AuthorizationCodeGrant) Execute(ctx context.Context, ssoCfg *SSOConfig,
 	if err := a.loginMAS(ctx, masSSOCfg); err != nil {
 		return err
 	}
+	a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggedIn"))
 	a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggedInMAS", localize.NewEntry("Host", masSSOHost)))
```

Comment:
```suggestion
	a.Logger.Debug(a.Localizer.MustLocalize("login.log.info.loggedInMAS", localize.NewEntry("Host", masSSOHost)))
```

While you are here, would you mind making this a debug message? I think the new ordering might be a little confusing, and the user does not really need to see two log in messages, now that I think about it :)